baker_syscall.go

```diff
@@ -0,0 +1,11 @@
+// +build linux darwin
```

Comment:
Shouldn't this be `!windows` instead in order to not exclude all oses where baker would currently compile?

Reply:
It seems better!